src/styles/utils/mixins.scss

```diff
@@ -0,0 +1,19 @@
+@mixin day-style {
```

Comment:
use placeholders in caseslike that https://medium.com/@linker21/scss-placeholders-vs-mixins-fe79b9dbb6d5